docs/user-guide/release-notes/2.12.x.rst

```diff
@@ -25,6 +25,9 @@ New Features
   the result on the orphan removal task. This count is broken down by content unit type. In
   pulp-admin, the result can be looked up with the ``tasks details`` command.
 
+* Failover events are now logged at the WARNING level with a message which explicitly states that
```

Comment:
Can this specify which components this is for. Also it would be good to use the term 'hot spare' and 'primary' somehow.